Tests/Unit/Hooks/Fixtures/TestingHookInterfaceReturnsModifiedValue.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Tests\Unit\Hooks\Fixtures;
+
+use OliverKlee\Seminars\Hooks\Interfaces\Hook;
+
+/**
+ * Valid test interface returning an array to use with the HookProviderTest.
+ *
+ * Will be accepted because it extends `Hook`.
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+interface TestingHookInterfaceReturnsModifiedValue extends Hook
+{
+    /**
+     * Gets called during HookProvider tests.
+     *
+     * @return bool
+     */
+    public function testHookMethodReturnsModifiedBool(bool $value): bool;
```

Comment:
For all the interfaces, please either drop all `@param` and `@return` that do not add anything more than the type declarations say, or please have them all. (Here, only `@return` is present, but not `@param`.)

Reply:
Sorry, didn't see that, too.